### PR TITLE
Toolbar

### PR DESCRIPTION
**Pull request by tlazarchuk**

### CODE REVIEW

File: src/app/pages/home-page/offer-list/offer-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (AlexanderBazukevich) [inline src/app/pages/home-page/offer-list/offer-list.component.ts:1]: in develop we have offer-list-page component. you dont need to commit this file. delete it locally and make push

File: src/app/shared/custom-components/toolbar/search/search.component.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  data!: SearchData[]; // Array of object, that contains a string data
+
+  stringData!: string[]; // Array of string, that was made from data
+
+  filteredData!: Observable<string[]>; // Array of filtred data, that application
+  // show user when he enter data
+  constructor(private searchService: SearchService) {}
+
+  ngOnInit() {
```
Comment (AlexanderBazukevich) [inline src/app/shared/custom-components/toolbar/search/search.component.ts]: add return type: void()

File: src/app/shared/custom-components/toolbar/tags/tags.component.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<div class="container-of-chips">
+  <mat-chip-list>
+    <div class="chips">
```
Comment (AlexanderBazukevich) [inline src/app/shared/custom-components/toolbar/tags/tags.component.html:3]: next time make service for getting tags and use ngFor

File: src/app/shared/mocks/mock-search-data.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { SearchData } from '../models/search-data';
+
+export const SEARCHDATA: SearchData[] = [
```
Comment (AlexanderBazukevich) [inline src/app/shared/mocks/mock-search-data.ts]: SEARCH_DATA

File: src/app/shared/custom-components/toolbar/toolbar.component.scss [hunk 5]
```diff
@@ -1,7 +1,8 @@
 .toolbar {
-  background-color: coral;
+  background-color: #fff;
   display: flex;
   height: 64px;
+  align-items: center;
 
   @media screen and (max-width: 768px) {
```
Comment (abarmina) [inline src/app/shared/custom-components/toolbar/toolbar.component.scss]: we have the variable "$tablet" in variables.scss, you can use this instead 768px (+ you will need a line 
```@import '/src/app/styles/variables.scss';```)

File: src/app/pages/home-page/offer-list/offer-list.component.ts (hunk 1 above)
Comment (tlazarchuk) [reply src/app/pages/home-page/offer-list/offer-list.component.ts:1]: Hmmm, it's don't work without these imports in my local branch...

File: src/app/shared/custom-components/toolbar/search/search.component.ts (hunk 2 above)
Comment (tlazarchuk) [reply src/app/shared/custom-components/toolbar/search/search.component.ts]: Ok

File: src/app/shared/custom-components/toolbar/tags/tags.component.html (hunk 3 above)
Comment (tlazarchuk) [reply src/app/shared/custom-components/toolbar/tags/tags.component.html:3]: Definitely!

File: src/app/shared/mocks/mock-search-data.ts (hunk 4 above)
Comment (tlazarchuk) [reply src/app/shared/mocks/mock-search-data.ts]: done

File: src/app/shared/custom-components/toolbar/toolbar.component.scss (hunk 5 above)
Comment (tlazarchuk) [reply src/app/shared/custom-components/toolbar/toolbar.component.scss]: done!

**Review by abarmina — APPROVED** [review]

**Review by AlexanderBazukevich — APPROVED** [review]

**Pull request merged**